lib/flipper/adapter_builder.rb

```diff
@@ -0,0 +1,44 @@
+module Flipper
+  # Builds an adapter from a stack of adapters.
+  #
+  #   adapter = Flipper::AdapterBuilder.new do
+  #     use Flipper::Adapters::Strict
+  #     use Flipper::Adapters::Memoizer
+  #     store Flipper::Adapters::Memory
+  #   end.to_adapter
+  #
+  class AdapterBuilder
```

Comment:
We are now moneta. 😆

We really should look at just using them at some point for adapters (or at least part of them).

Reply:
I'm all in favor of that.